django/publicmapping/static/js/mapping.js

```diff
@@ -2465,11 +2465,57 @@ function mapinit(srs,maxExtent) {
             return rules;
         };
 
+        var getDistrictShadingRules = function() {
+            districts = [
+                { id:1, name:'District 1', color: '#d74e36'},
+                { id:2, name:'District 2', color: '#e38740'},
+                { id:3, name:'District 3', color: '#e9d579'},
+                { id:4, name:'District 4', color: '#56875a'},
+                { id:5, name:'District 5', color: '#46618b'},
+                { id:6, name:'District 6', color: '#6f5b76'},
+
+                { id:7, name:'District 7', color: '#d74e36'},
+                { id:8, name:'District 8', color: '#e38740'},
+                { id:9, name:'District 9', color: '#e9d579'},
+                { id:10, name:'District 10', color: '#56875a'},
+                { id:11, name:'District 11', color: '#46618b'},
+                { id:12, name:'District 12', color: '#6f5b76'},
+
+                { id:13, name:'District 13', color: '#d74e36'},
+                { id:14, name:'District 14', color: '#e38740'},
+                { id:15, name:'District 15', color: '#e9d579'},
+                { id:16, name:'District 16', color: '#56875a'},
+                { id:17, name:'District 17', color: '#46618b'},
+                { id:18, name:'District 18', color: '#6f5b76'}
+            ];
+
+            rules = [];
+
+            districts.forEach(function(d) {
```

Comment:
It doesn't really matter, but this is a prime use case for `Array.prototype.map`:
```
            return districts.map(function(d) {
                return new OpenLayers.Rule({
                    filter: new OpenLayers.Filter.Function({
                        evaluate: function(feature) {
                            return feature.district_id === d.id;
                        }
                    }),
                    symbolizer: {
                        fillColor: d.color,
                        fillOpacity: 0.5,
                        strokeColor: '#000000',
                        strokeWidth: 2,
                        strokeOpacity: 0.67
                    }
                });
            });
```